test-requirements.txt

```diff
@@ -94,8 +75,9 @@ parso==0.8.3
     # via jedi
 pathspec==0.12.1
     # via black
+pip==24.0
+    # via pip-tools
```

Comment:
I think uv should respect the pip-tools thing where it doesn't specify `pip`?

Tbh idk why pip-tools doesn't specify `pip` and whether it's relevant for us.

Reply:
> `pip-tools` is going to change their default soon to include it https://github.com/astral-sh/uv/issues/1353